extras/AFC.py

```diff
@@ -16,7 +15,9 @@ def __init__(self, config):
         self.printer.register_event_handler("klippy:connect",self.handle_connect)
         self.SPOOL = self.printer.load_object(config,'AFC_spool')
         self.ERROR = self.printer.load_object(config,'AFC_error')
+        self.IDLE = self.printer.load_object(config,'idle_timeout')
         self.gcode = self.printer.lookup_object('gcode')
+        self.STATUS= self.IDLE.state
```

Comment:
Since this will always return the same state and not update, this line can be removed